library/src/layout/table.rs

```diff
@@ -298,7 +298,7 @@ impl LocalName for TableElem {
             Lang::SLOVENIAN => "Tabela",
             Lang::SPANISH => "Tabla",
             Lang::UKRAINIAN => "Таблиця",
-            Lang::ENGLISH | _ => "Table",
```

Comment:
I think it's clearer with the explicit `Lang::ENGLISH`

Reply:
We can disable `clippy::wildcard_in_or_patterns` or split it into two lines:
```
Lang::ENGLISH => "Table",
_ => "Table"
```

Reply:
I would prefer disabling this clippy lint globally as whenever I do this, it is for a reason. Duplicating the string isn't that nice.

Reply:
@laurmaedje Done